chart/templates/_helpers.tpl

```diff
@@ -272,6 +272,10 @@ env:
   value: {{ $tracing.samplerType }}
 - name: JAEGER_SAMPLER_PARAM
   value: "{{ $tracing.samplerParam }}"
+{{- if $tracing.tags }}
+- name: JAEGER_TAGS
+  value:  "{{ $tracing.tags }}"
+{{- end }}
```

Comment:
This will break with the installer and it's unlikely we'd introduce this as configuration value.
We could add them in code though.